### DISCUSSION TITLE
Notary selection and incentives for v5 elections

### DISCUSSION THREAD

**Discussion by raghavrmadya** [filecoin-project/notary-governance#961] [Ideas]
It has been widely discussed that one of the underlying challenges in addressing DC abuse is notary compensation. Inherently, we have seen a significant overlap in the SP and clients roles notaries play in the current system which makes it extremely challenging to separate the due diligence duties from data onboarding and storage functions in the network. 

As such, this discussion invites thoughts and suggestions on compensating notaries for their time and implementing such change as soon as the V5 election

**Comment by raghavrmadya** [comment]
My own opinion (not PL) is that the community must not allow the election of active SPs DPs and Clients in the v5 notary cycle. 

Instead, finding ways to enable anyone to become a notary, as suggested [here](https://github.com/filecoin-project/FIPs/discussions/774) is the most reasonable path while developing mechanisms to compensate notaries for their time doing due diligence. 

Some suggestions on compensating notaries include but are not limited to - 

1. Payment by the client that wishes to receive DC 
2. Enabling staking to proportionate to DC requested allowing anyone to become a notary and getting rid of an "election" altogether
3. Community grant was given by FF to compensate "elected" notaries to further it's mission

Other ideas are welcome

**Reply by dcasem** [reply]
This is **EXACTLY** what I warned about on the [July 25th Notary Goverance call at 52:50. ](https://www.youtube.com/watch?v=QciYj-G5rdU&t=&start=3170)

It also is inconsistent with the most recent messaging around people reapplying if they were inactive (as recently as yesterday).

Honestly, it's hard to believe that anyone can believe that the Filecoin network benefits from an increasingly centrally controlled system where the rules constantly change at the whim of the Fil+ Governance team, despite promises otherwise. 

I encourage disaffected notaries to engage in the discussion on [774.](https://github.com/filecoin-project/FIPs/discussions/774)

**Reply by cryptowhizzard** [reply]
It seems you did not read RG’s text thoroughly. RG suggested that everyone must be able to become a notary, thus that enables decentralisation, however he suggests to attempt to block conflicts of interest. Ain’t that a good thought worth discussion?

I don’t mind activism, but you sound like you want to block every attempt to get things straight in the Fil+ community despite the outcome of 774. That sure is not in the benefit of this network.

**Reply by dcasem** [reply]
What did I miss from this @cryptowhizzard 

>My own opinion (not PL) is that the community must not allow the election of active SPs DPs and Clients in the v5 notary cycle.

>That sure is not in the benefit of this network.

Stopping the centralization of the power of Fil+ is most definitely to the benefit of the network. We already have a permissioned chain. Now we want a more centrally controlled permissioning of that chain? How exactly does that benefit the network?

**Reply by cryptowhizzard** [reply]
@dcasem like i suggested. RG suggested that we should look for independent notary’s not affiliated or SP / DP to avoid conflicts of interest and make it healthy. Can you give that thought please? 

Second, if we are on the road of 774 it might be usefull to clear the bumps on the road and prepare for whatever is to come. Thinking of a better notary system does not harm anyone and helps for whatever path Fil takes in the future

**Reply by dcasem** [reply]
@cryptowhizzard I'm all for iterating. For example, I'm supportive of paying notaries for their work. In fact, I've suggested it multiple times, both on the calls and in writing in the fil-plus channel. I'm vehemently against limiting who can participate as a notary, especially when we told people who were just removed that they could re-apply. 

>I don’t mind activism, but you sound like you want to block every attempt to get things straight in the Fil+ community despite the outcome of 774. That sure is not in the benefit of this network.

Even if [774](https://github.com/filecoin-project/FIPs/discussions/774) is to be implemented, contrary to your statement, I have encouraged [changes to the notary program](https://github.com/filecoin-project/FIPs/discussions/774?sort=new#discussioncomment-6649343).

**Reply by cryptowhizzard** [reply]
@dcasem that is good to hear. So let’s figure out how we can come to optimal performance in the situation that 774 does not make it (perhaps) and prepare for the fact that Fil+ will stick around for a while, because something needs to happen.

The optimal situation would then be that we eliminate abuse of datacap and focus on real use cases.

Eliminating abuse is time consuming. We need people to cooperate to get that done thus we need more notary’s doing DD. I agree with you on that. The Q is how we motivate them to do the right things in the benefit of the network while we evolve to a healthier situation. Second we need to set motivation for people who don’t do the right thing and harm us all.

What are your thoughts on that?

**Reply by kernelogic** [reply]
I would say if a person that is not SP, DP, or Client, he/she doesn't really have the filecoin knowledge qualification to be a notary. Who's left would be interested in this case, token holders? 

You'll have to come up with a course or something with certification to get such "newbie" to be a notary. Thinking that as a bar exam in lawyers world.

**Reply by raghavrmadya** [reply]
@kernelogic , while I agree that an "outsider" might not have filecoin knowledge, I don't think I agree they don't have qualification. This becomes a chicken and egg problem then. There is a learning curve, arguably steep, to become an SP, DP and Filecoin client as well and I don't think we should disqualify "newbies" to be a notary. If the community thinks this is a good direction to explore, putting together resources is not hard or necessarily challenging. The ultimate goal is anyway (in my mind atleast) to have smart contracts be notaries, not humans. To get their, we need a lot of "training data" from active notaries like yourself and @cryptowhizzard and a lot more. 

We have two dev summits coming up. If we really want to make accretive improvements while 774 takes shape, I would side with not limiting notary functions to SP, DP and Client and yes, my most ideal scenario, albeit a bit wishful, is to reach threshold of "outsider" notaries who are not current SP, DP, and Client

**Reply by nai2283** [reply]
So the question becomes why outsiders bother to conquer the learning curve to become notary if there is no incentives or interests to do so? If they have, then bias and conflict interests of those notaries will be inevitable, just like the existing notaries, catch 22?

**Reply by The-Wayvy** [reply]
@cryptowhizzard 

> The optimal situation would then be that we eliminate abuse of datacap and focus on real use cases.
> 
> Eliminating abuse is time consuming. We need people to cooperate to get that done thus we need more notary’s doing DD. I agree with you on that. The Q is how we motivate them to do the right things in the benefit of the network while we evolve to a healthier situation. Second we need to set motivation for people who don’t do the right thing and harm us all.
> 

This is too subjective.

What is real? What is the right thing? What is healthy? What is abuse?

Paying users can pay and everyone else can get lost.

**Reply by dkkapur** [reply]
@nai2283 I think the literal point of @raghavrmadya's suggestion was that they need to be compensated for this to be an option. so you're both on the same page there.

**Reply by dkkapur** [reply]
@The-Wayvy - agree, i think deals that are paid for are a good indicator of utility served. there are probably other measures too though. how do we get to objectively measuring/tracking this if payments are sometimes off-chain / in fiat?

**Reply by The-Wayvy** [reply]
@dkkapur 
@raghavrmadya 

1

Maybe there are objective measures of usefulness other than $ paid.

Let’s see if someone brings them up.

2

Off-chain payments are unnecessary.

If an app is real and useful people will just set up a wallet and pay. It’s pretty easy. That’s what is done on Ethereum. 0 problems. Just add USDC / USDT to Filecoin. This is a good ecosystem project to fund

If a user doesn’t want to use a crypto wallet they are probably not good customers.

3

From a BD perspective, I share views with
@mjroddy . Hope you guys talk to him more.

Better Aggregation is the lowest hanging fruit.

The big clients will only come at the end.

CERN, Lockheed, etc are just photo ops / marketing stunts.

This judgement is based on my observations of BTC and ETH’s development.

**Reply by The-Wayvy** [reply]
@dkkapur 
off-chain payments is the shadiest thing I've ever heard in my life

**Comment by cryptowhizzard** [comment]
I would like to suggest notary teams where each team consists of 1 tech person, 1 commercial person and one communication person.
I would advise to have people with tech knowledge to be eligible for the technical role despite they might work for an SP or DP so he / she can feed proper input to the communication / commercial persons.

Staking and compensation are good motivations to eliminate abuse. F.e.

Staking amount shall be payed by the LDN applicant. He/she can get their funds back after the LDN is proper. 
- Data is stored with SP's who applicant confirmed in application.
- Data samples are downloaded and checked and match the dataset a applicant is storing.
- KYC done.

To avoid collusion notary compensation should be done by PL/FF. Compensation is released when:
- LDN signed where data is stored with SP's who applicant confirmed in application.
- LDN signed where data samples are downloaded and checked and match the dataset a applicant is storing.
- LDN signed in line with #922

**Reply by The-Wayvy** [reply]
This will make the abuse more sophisticated.

If there's an incentive for it to happen, it will.

Build a protocol, not a bureaucracy.

No one wants to invest in IRS Chain.

**Reply by kernelogic** [reply]
Do you mean divide 90 notaries to 30 groups? Or each notary must consist minimum 3 people (so that individual contributors like me do not qualify anymore) ?

**Comment by raghavrmadya** [comment]
@dcasem, clarifying that everyone here has the ability to have an opinion. I'm merely expressing that the community consider NOT electing notaries from a pool of active SPs DPs and Clients in the v5 notary cycle and if you are objecting to my use of the word "must", I'm allowed to campaign for things I believe in just like you.

**Reply by dcasem** [reply]
> clarifying that everyone here has the ability to have an opinion

I'm well aware. However, you shouldn't be surprised when I call you out for being entirely inconsistent with what you guys said on the last two calls I attended (feel free to review the recordings).

I'm sorry, but your words come from a place of authority and therefore are subject to increased scrutiny.

**Reply by dkkapur** [reply]
>I'm well aware. However, you shouldn't be surprised when I call you out for being entirely inconsistent with what you guys said on the last two calls I attended (feel free to review the recordings).

@dcasem what specifically did you find to be contradictory? just so that we're discussing the same thing. thanks.

**Reply by dcasem** [reply]
@dkkapur I outlined this [here](https://github.com/filecoin-project/notary-governance/discussions/961#discussioncomment-6683449). 

Listen to what was said on this call vs. what is being suggested in this discussion by @raghavrmadya.

**Comment by The-Wayvy** [comment]
Oppose option 3.

Filecoin must be Permissionless.

https://github.com/filecoin-project/FIPs/discussions/774#discussioncomment-6631403

@momack2 @raghavrmadya 

Molly, tell Marko Vukolic why he's wrong.
Tell him why he is not aligned with the mission.
Tell him why he is missing the obvious context.
Please, we are all waiting.

**Reply by dkkapur** [reply]
@The-Wayvy can we keep the convo scoped to the topic? thanks for engaging.

**Comment by psh0691** [comment]
I hope FIL+ will develop from the next notary selection.
This is the situation of a system member of FIL+ that I have watched so far.


Notary
- There were 88 fourth notaries, but there was an increase in inactive notaries, and discussions were made to remove them. 
- Until now, only the notary's responsibility has been emphasized, but there has been no reward. 
- Lack of expertise and differences in perspectives based on mutual interests made it easy for conflicts to arise during the verification process. 
 - Training was insufficient after choosing a notary.
 - Notaries are intertwined with SP and DP, and applications that are not interested are often ignored for a long time.   

Client.
- Many data owners and creators are ignorant of the Filecoin network.
- It has many advantages and disadvantages over traditional cloud systems, so it is reluctant to use them.
 - Most people apply for DC as a business partner for SP, but validation and assignment can take a long time.

SP
- It's the biggest power of the Filecoin network, but it's gradually losing power.
- Real estate, computer equipment, technology, and manpower have been put in, but it is difficult to maintain the status quo.
Small and medium-sized SPs are losing ground due to the economic recession, falling coin prices, rapid reduction of compensation, and businesses to secure data.

**Reply by kernelogic** [reply]
Well said @psh0691

**Reply by dkkapur** [reply]
@psh0691 agree. any suggestions for how we should continue evolving the notary selection process to help with these challenges?

**Reply by 1ane-1** [reply]
Support！

**Reply by psh0691** [reply]
@dkkapur 

Proposal for selection of notaries

1. Provide incentives
- Subject of provision: FF, PL
- It is unfair for the client or SP to bear the cost of due diligence among FIL+ members
- SP and data retention clients can be seen as essential members and major customers of FIL+.
- However, collateral (staking) is worth considering so that clients are not caught up in fraud/abuse when applying for DC.

2. Reestablishment of Notary Selection Criteria
- The activity of the notary in the previous round should be actively reflected, and the notary of the active activity should be first mentioned. The experience of due diligence is another intangible asset of FIL+.
- The selection criteria for notaries of FIL holdings seem unfair, so it would be nice to modify or delete them.

3. Adjustment of Notary Activities
- The period of activity of the selected notary shall be indefinite.
-However, if you want to quit, it should be handled in some form of expression of intention, 
- Regulations for the removal of inactive notaries are prepared and processed.
- Additional selection shall be made in proportion to the amount to be due diligence.

4. Feedback from Notary Activities
- In June, it was personally stimulated by receiving feedback from notary activities. 
- Regular and continuous feedback helps notary activities.

5. Remediation of unclear regulations/guidelines
- Controversial regulations/guidelines should be revised and supplemented as soon as possible.
- For example, specific guidelines such as data merging problems and data distribution problems should be prepared.
- One SP may have multiple nodes in one region or in multiple regions. Alternatively, multiple SPs can use a data center in one area. Regulations on whether to view a country as a region or a city as a region should also be established.

That's what I thought personally. Please refer to it and if you have any better suggestions, it would be great.

**Comment by nicelove666** [comment]
I think this is a good proposal, which shows that each of us is working hard to achieve a better FIL+. Below, we discuss two issues:

First, if we do not choose from active SPs DPs and Clients, how should we choose? Can you list more detailed selection criteria. In my opinion, as long as quantifiable standards are listed and accepted by the community, this suggestion can be implemented. Any discussion takes time, and we should give ample time and patience to the development of standards.

Second, perhaps this is a dialectical relationship, not all active SPs DPs and Clients are abusing DC, but those who abuse DC are often very active. We can choose a "novice" as a notary, but when he becomes a notary, he may change from an "outsider" to an "insider".

People who abuse DC are often powerful people. When you give anyone power, he may abuse it.

**Reply by dkkapur** [reply]
i think its very hard to distinguish individuals from other roles they may play in the ecosystem. everyone can assume multiple IDs, hide behind anonymity, or partner under the table with other entities. as a result, i think its better to build systems around notaries that push them to serve the network better with compensation rather than need to be served themselves by collusion.

**Reply by The-Wayvy** [reply]
@dkkapur 
You see the problem.

But I don’t think FF paying notaries solves it.

It just makes it easier for FF to commit fraud. (Could be pilot programs with onboarding of real data whose storage will never be paid for by its owners). 

If notaries are not compensated by FF then they will work with SPs.
If notaries are compensated by FF then they will work with FF.


As long as notaries have the power to hand out 10x multipliers they will be targeted.

3

This is just too much power to put in the hands of any individual.

Community members will have to police notaries. It’s a waste of time and energy.

**Comment by Destore2023** [comment]
> I hope FIL+ will develop from the next notary selection. This is the situation of a system member of FIL+ that I have watched so far.
> 
> Notary
> 
> * There were 88 fourth notaries, but there was an increase in inactive notaries, and discussions were made to remove them.
> * Until now, only the notary's responsibility has been emphasized, but there has been no reward.
> * Lack of expertise and differences in perspectives based on mutual interests made it easy for conflicts to arise during the verification process.
> * Training was insufficient after choosing a notary.
> * Notaries are intertwined with SP and DP, and applications that are not interested are often ignored for a long time.
> 
> Client.
> 
> * Many data owners and creators are ignorant of the Filecoin network.
> * It has many advantages and disadvantages over traditional cloud systems, so it is reluctant to use them.
> * Most people apply for DC as a business partner for SP, but validation and assignment can take a long time.
> 
> SP
> 
> * It's the biggest power of the Filecoin network, but it's gradually losing power.
> * Real estate, computer equipment, technology, and manpower have been put in, but it is difficult to maintain the status quo.
>   Small and medium-sized SPs are losing ground due to the economic recession, falling coin prices, rapid reduction of compensation, and businesses to secure data.

100% agree with this.

There is a DP role that helps clients with data process and deal-making, he/she is an important bridge between clients and the Filecoin network.

The status quo is that one person plays more than 2-3 roles, which is probably a big reason why FIL+ has gone to nowadays.

Regardless of the selection criteria for V5, the above should never happen again. @raghavrmadya

**Reply by The-Wayvy** [reply]
As @nicelove666 said, you can’t keep notaries from developing relationships with SPs. Option 3 will change nothing. Fraud will remain until the protocol is Permissionless.

**Comment by Janicelu1** [comment]
I don't think there is a need to elect V5 notaries because future notaries would be redundant. Anyone who get the normal reports will automatically get the quota for the next round.

**Comment by nicelove666** [comment]
Let's move on to:

1. The notary pledges to sign. If the SP and the client are honest, the notary will be rewarded

2. If either the SP or the client is fraudulent, the notary will be punished, the FIL pledged for the signature will be confiscated, the notary will be disputed, and the notary may be disqualified.


If so, we run into three problems:

1. How do we judge the honesty or fraud of customers and sps? What are the key indicators?

2. We need to conduct a unified inspection, download data, analyze data, and evaluate data for all LDNs. This is a very fortunate matter. Who will do this matter, and can it be completely fair and just?

3. Where does the notary’s pledge come from? Is the notary capable of paying such a large amount of pledge? Then, in this case, the notary will inevitably cooperate with the sp and let the sp pay the pledge.

4. Notaries who are unable to provide pledges will become silent, and even if there are 200 notaries in FIL+, only 20 active notaries who are interested in SP will sign in the end.

**Reply by cryptowhizzard** [reply]
2. Notary’s will be rewarded for their work. They can hire people to do inspections or do it themselves.

3. pledge comes from the DP who asks to sign. 
DP’s will become more motivated to screen SP’s and their reputation and stop hiding from their responsibility. DP’s will engage in contract’s with SP’s and probably forward the collatoral requirement partially to the SP. 
Notary’s will also start making contracts with DP’s. Clear agreements are made about what / where and when between DP and notary’s.

1. sample retrievals are part of the game already. If the bot can make them available publically to check for everyone this problem is solved. The notary only has to ask where he can find and check the file in the original set he / she  is looking at. Without a successfully sample subsequent allocations should not be signed.

 motivations for slash of pledge:

- pledge shall be slashed if the data a participant is storing is not in the dataset indicated
- Pledge shall be slashed if their are signatures without retrieval samples from the bot.

**Reply by The-Wayvy** [reply]
If someone wants to use Filecoin, they can pay for it. 

We don’t need freeloaders. They contribute nothing.

**Comment by Bennyyangpu** [comment]
Agree with @nicelove666, We cannot solve an old problem and bring in a new one at the same time.

**Comment by kernelogic** [comment]
Taking consideration where this is going, I am going to propose the following:
1. Avoid conflict of interest is the Number 1 priority.
2. Dismiss every v1234 notary and start over with a smaller number, for example 20.
3. New v5 notaries must NOT: 
3.1 personally have open DC or will apply new DC, 
3.2 personally seal DC, 
3.3 owning or employed by entities that have open DC or will apply DC, 
3.4 owning or employed by entities that will seal DC, 
3.5 employed by PL, 
3.6 employed by FF.
4. Go through comprehensive training for Fil+ related, ethical and technical knowledge and pass exam.
5. Sign or assume NDA is in place to access all stored data.
6. Be provided with proper access with internet IN ALL MAJOR REGIONS to test retrievals (for example AWS VMs in different regions).
7. KPI of reviewing minimum 10 LDNs per week or 40 LDNs per month. Randomly assigned by bot. Currently there are 378 open LDNs, 20 notaries can review 800 LDNs per month. Missing KPI subject to immediate termination. Every review must include more than 2 successful retrievals and verify the content, reject the proposal if
7.1 Data is too small for the sector size
7.2 Data is encrypted without providing key
7.3 Data cannot be opened
7.5 Data has excessive duplications
7.6 Data is artificially enlarged 
8. **Since no one has proposed a concrete number, I'm gonna say as a starter, compensation of 250 Fil per week, or 1000 Fil per month, paid by FF, not by client, not by SP. Zero tolerance on bribery.**


![straight-to-jail-crime](https://github.com/filecoin-project/notary-governance/assets/3711825/630af83f-eb47-4d89-b6ed-ab18a0829f2e)

**Reply by 1475Notary** [reply]
Agree.

**Reply by herrehesse** [reply]
@kernelogic Distribute them geographically across the globe? 5 per continent?

**Reply by herrehesse** [reply]
3.1/2/3/4 can be easily spoofed by running a new node on VPN, create a throwaway github account and fully self-deal? All these questions need "trust" and if we learned one thing is that that trust is abused to its maximum.

**Reply by herrehesse** [reply]
All other points are extremely valid. Fully agreed.

**Reply by The-Wayvy** [reply]
@herrehesse 
@raghavrmadya 
@dkkapur 
@kernelogic 

Asia has over 4 and a half billion people and 40 trillion GDP.

Europe has 0.75 trillion trillion people and 25 trillion GDP

No way Europe gets the same number of notaries as all of Asia.

**Reply by herrehesse** [reply]
Absolutely, it's imperative that Europe, the USA, and Australia receive an equitable share of notaries as Asia does. In the previous iteration, a significant proportion of notaries originated from China, and even the majority of spurious accounts from other areas were essentially tied to Asian miners/clients/notaries. In hindsight, we now recognise that virtually all of the misconduct stemmed from this specific region and it's notaries. **We mustn't allow history to repeat itself.**

Our focus should be on attracting REAL users and businesses that are eager to participate in the network, allocating resources based on this criterion rather than raw computational power. As you articulated earlier, assessing GDP reveals a remarkable balance across all three continents, a factor of greater significance in my opinion.

Fake users, fake businesses, and the prevalent fabrication of account claims regarding data cap have regrettably become all too familiar over the past 8 months. However, it's crucial to emphasise that this isn't where our attention should be fixated. Instead, we must direct our efforts towards fostering genuine network participation and maintaining the integrity of the system. 

In my opinion, a configuration of 5 notaries per continent adhering to the aforementioned guidelines proposed by @kernelogic is entirely aligned with the principles of effective program implementation.

**Reply by The-Wayvy** [reply]
5 per continent is not equitable at all 

Australia and NZ have a combined GDP of 2 trillion.

If we’re going to go with a quota then at least weigh distribution by population or GDP or RBP or TVL locked into apps produced by that region etc.

A better system would be an admissions test. Highest scorers get in. Obviously to be eligible for the test you’d have to meet other criteria like not being an FF employee.

Also I know it’s a bit off topic, but users who don’t pay are not real. It doesn’t matter what kind of fancy documents they provide / fabricate. Users must pay.

**Comment by licornduan** [comment]
True, false, cold and hot are behavioral data created by different SP situations. Storing data is SP's choice and is influenced by policies, regulations and the industry environment. Data retrieval is a requirement of the upper-level data demand scenario, we can create a new project fil++, open source programs reference data CID from IPFS, give the corresponding datacap quota reward. For example, if the small program or portal internal data reference connection comes from the corresponding SP supplier, the corresponding SP supplier will be provided with the corresponding SP credit reward because of its contribution to the development of ecology.

**Comment by herony-fil** [comment]
Is it possible to set up a reward and punishment mechanism to automatically constrain the behavior of notaries? Like set up a reward pool? For example, a certain amount of FIL will be collected from all notaries as pledges at the beginning of a period(like the beginning of a month), and the FIL pledged by inactive notaries will be redistributed in proportion to the most active notaries (can be decided by voting, etc.) at the end. The rest of the pledges can be carried over to the next period. If notaries no longer want to continue to play this role, they may also have a withdrawal option to take back the FIL coins they pledged.

**Reply by The-Wayvy** [reply]
@herony-fil 
This will not help Filecoin.
Real users pay. We don't need notaries doing KYC and whatnot.

**Reply by baseonejj** [reply]
Where are the real users?

**Reply by herrehesse** [reply]
@baseonejj Very few are authentic. Among the 2000 LDN applications, genuine ones make up less than a few percent.

**Reply by The-Wayvy** [reply]
@baseonejj 

The real users are miners 
Because they pay tx fees 
That is reality 

And a few people on the FVM

All the freeloading ‘data clients’ are completely fake at worst or marketing stunts at best

**Comment by kevzak** [comment]
I'm with @herony-fil on the reward and punishment mechanism, but I think we could make this notary selection and incentive process much easier.

**Anyone should be able to be a notary.**

Here are the requirements:
- Stake 2500 FIL (or whatever amount works best for the community)
- Complete a KYC check matching your ID to your GitHub account

That's it.

And there are two rules as a notary:
- If you approve an application and the data onboarding is completed successfully, you'll get x FIL for your work as a notary on the application (let's say 20 FIL)
- However, if an application is shutdown for any violation of requirements, you get slashed and the collateral FIL is burned (or some portion depending size of application)

Resulting behaviors we are looking to create:
- The community getting behind successful onboarding of useable datasets (Good Deal Making)
- The community getting away from supporting bad deal making

The catch with this approach is that clear, objective, measurable requirements need to be implemented first in the areas of: 
- Dataset distribution 
- Data Preparation
- Retrievability 
- Data sampling 

We need to continue discussions and enhancements in these areas. If we can make quantifiable on-chain data 'indicators' part of the approval process, the notary job can become more of a checks and balances role than as a guardian of DataCap.

**Reply by The-Wayvy** [reply]
@kevzak 
@raghavrmadya 
@dkkapur 
@jnthnvctr 
@herony-fil 
@kernelogic 

This is much better than what we have now 👍

But if the staking and slashing mechanism works then there’s no need to do KYC 

A notary’s identity has nothing to do with the criteria you listed for ‘realness’ of a deal.

I thought we had reached consensus on the need for objectivity and permissionlessness.

**Reply by jnthnvctr** [reply]
thanks for the tag! think this is a great step - tho shouldnt FIL requried as stake be sized to amt of datacap being allocated (or at least data cap over some period of time?) 
- maybe people can delegate stake to the notary if theyre highly trusted (?) 
- not sure if 2500 is enough if someone is allocating pibs of datacap

+1 to wavvy re kyc, im not sure how it helps? 
+1 that this is an improvement on the current system - but not really solving for the permissionless problem

**Reply by kevzak** [reply]
Completing KYC as described above does not mean to know the person behind the GitHub account. Their identity is actually never revealed. 

Completing KYC for Fil+ only means that a real person:
- Verified their identity as a real unique person
- and that verified person is associated to a specific GitHub ID
- and that specific verified person cannot be associated with another GitHub ID

To your point, we may not need notaries to complete KYC for the process to operate successfully. However, I still think it adds value to verify GitHub IDs, for all users not just notaries, to ensure a specific verified real human is associated with one account.

**Reply by nicelove666** [reply]
Thanks for your message, it makes everything so much easier. Anyone can become a notary. What we need to pay attention to is the behavior of the notary, not the organization or other interest institutions represented by the notary.

**Reply by herrehesse** [reply]
For more than six months, nobody paid enough attention, which is why we're in this mess. And fake attention from abusers doesn't count.

**Reply by Stanton15** [reply]
Staking and slashing mechanism would not help solving the current crisis of fil+, as the hardest part is the conclusion determination, and it's not a question of whether the notary has fil to stake or not.

And I can assert that the more complex the mechanism, the more potential for abusive behavior. For a lot of people who take DC as a business, it's a simple math question whether to pledge or not.

**Comment by MRJAVAZHAO** [comment]
- About the notary incentives

My suggestion is to improve the notary mechanism that combines tools, tasks and incentives related to Trust&Transparency.

1. Bots and tools cooperate with each other to conduct KYC verification and data storage inspection. The bots add lables based on the results of the tool, KYC complete and abnormal data retrieval.

2. Issues with problematic labels will be randomly sent to unrelated notaries, who will complete the inspection of applications and receive incentives.

3. If the notary does not handle the issues for more than a period of time, the notary's right will be temporarily frozen. The notary will submit an application to the community, and the community will review and confirm whether to cancel the freeze.

Based on the above mechanism, we need to further improve the community rules and complete the tool development as soon as possible. Avoid uneven workload distribution, discrimination, monopoly and other situations.


- About the fifth round of notary elections

The rules and order of the community were in chaos for a period of time. If notaries who are willing to continue due diligence, they can form the fifth round of notaries team. At present, the existing notaries have a certain understanding of the community and rules, which reduces the education cost. If there is no completion of the task and notaries removal, the community conduct the phased of new notaries expansion.

**Comment by woshidama323** [comment]
> * However, if an application is shutdown for any violation of requirements, you get slashed and the collateral FIL is burned (or some portion depending size of the application)

IMO, Why not back the slashed FIL to the reward pool? 

Should we have clear rules or workflow for judging if an application should be shut down or continue or DataCap removing?

**Comment by 1475Notary** [comment]
Applications should be judged by objective codes or reports and not overly rely on supposedly upstanding notaries.
It seems to me that incentives and pledging mechanisms have the potential to make abuse more common. 

I agree to a large extent with the community must not allow the election of active SPs DPs and Clients in the v5 notary cycle.

**Comment by The-Wayvy** [comment]
I’m still waiting for someone to propose a measure of usefulness other than willingness of the user to pay.

@dkkapur 
@raghavrmadya

**Comment by VenusOfficial** [comment]
0.0
